src/bidmanager.js

```diff
@@ -70,10 +70,13 @@ function getBidSetForBidder(bidder) {
 exports.addBidResponse = function (adUnitCode, bid) {
   if (bid) {
     //first lookup bid request and assign it back the bidId if it matches the adUnitCode
-    let bidRequest = getBidSetForBidder(bid.bidderCode).bids.find(bidRequest => bidRequest.placementCode === adUnitCode);
-    if(bidRequest && bidRequest.bidId) {
-      bid.adId = bidRequest.bidId;
+    if (!bid.adId) {
+      let bidRequest = getBidSetForBidder(bid.bidderCode).bids.find(bidRequest => bidRequest.placementCode === adUnitCode);
```

Comment:
Bid responses vary and there was initially not much standardization as Prebid is a wrapper around existing header bidding tech. It has since become more evident that there should be more standards to improve header bidding solutions generally, and matching requested bid IDs with bid response IDs is certainly a standard we'd like to see adopted.